canonical/lxd-ui
# feat: refactor side panels and enabled overlay panels

## Done

- Removed the `DetailPanel` and `Aside` components and refactored them into a compound component `SidePanel`
- Introduced overlay styling variable for the side panel
- Updated `InstanceDetailPanel` and `ProfileDetailPanel` to use `SidePanel`

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - Ensure that instance detail panel and profile detail panel still works as expected.
